src/components/TeamGenerator.js

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from 'react';
+import { pickTeams } from '../services/generateTeams';
+import ImportPlayerList from './ImportPlayerList';
+import TeamCompCard from './TeamCompCard';
+
+const TeamGenerator = () => {
+	const [playerList, setPlayerList] = useState([]);
+	const [teamA, setTeamA] = useState([]);
+	const [teamB, setTeamB] = useState([]);
+	const removeAble = (player) => {
+		setPlayerList(playerList.filter((p) => p != player));
+	};
+
+	return (
+		<div className='flex flex-col items-center'>
+			<ImportPlayerList
+				props={{
+					callback: (importPlayers) => {
```

Comment:
create the function outside the prop,
change the name of this callback to a proper name, for me i know its a callback no fucking clue what's the purpose